Userland/Libraries/LibWeb/FileAPI/Blob.cpp

```diff
@@ -166,7 +167,7 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<Blob>> Blob::create(JS::Realm& realm, Optio
 
         // 2. Convert every character in t to ASCII lowercase.
         if (!type.is_empty())
-            type = options->type.to_lowercase();
+            type = TRY_OR_THROW_OOM(vm, options->type.to_lowercase());
```

Comment:
`String::to_lowercase()` is _not_ ASCII lowercase, use the helper in `Web::Infra`:

https://github.com/SerenityOS/serenity/blob/76b71fcb75d17815354529e1916145fb56d736eb/AK/String.h#L103-L107